openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -774,6 +807,10 @@ def setUpClass(cls):
             'storage_class2', cls.storage_classes.get('block_storage_class'))
         cls.secret_type = "gluster.org/glusterblock"
 
+        cls.registry_sc = cls.storage_classes.get(
+            'storage_class4', cls.storage_classes.get('registry_block_storage_class'))
+        cls.registry_secret_type = "gluster.org/glusterblock"
```

Comment:
"gluster.org/glusterblock" is not valid.

Reply:
same as above comment